File: App/abc/app/src/main/java/com/example/unit_converter/MainActivity.java
package com.example.unit_converter;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    private Button button;
    private TextView textView;
    private EditText editTextTextPersonName;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        button=findViewById(R.id.button);
        textView=findViewById(R.id.textView);
    editTextTextPersonName=findViewById(R.id.editTextTextPersonName);
    button.setOnClickListener(new View.OnClickListener() {
        @Override
        public void onClick(View v) {
            Toast.makeText(MainActivity.this, ":)", Toast.LENGTH_SHORT).show();
            String s=editTextTextPersonName.getText().toString();
            int kg=Integer.parseInt(s);
            double pound=2.205*kg;
            textView.setText("The corresponding value in pounds is : "+ pound);

        }
    });
    }
}

File: App/QuizApp/app/src/main/java/com/example/quizapp/MainActivity.kt
package com.example.quizapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        window.decorView.systemUiVisibility=View.SYSTEM_UI_FLAG_FULLSCREEN

    val tv2:Button=findViewById(R.id.btn_start)
        tv2.setOnClickListener {
            val tv3:EditText=findViewById(R.id.et_name)
            if(tv3.text.toString().isEmpty())
            {
                Toast.makeText(this,"Please enter your name",Toast.LENGTH_SHORT).show()
            }
            else
            {
                val intent=Intent(this,QuizQuestionsActivity::class.java)
                startActivity(intent)
                finish()
            }

        }
    }
}